src/compile/layer.ts

```diff
@@ -184,12 +189,13 @@ export class LayerModel extends Model {
     return applyConfig({}, cellConfig, FILL_STROKE_CONFIG.concat(['clip']));
   }
 
+  // TODO: Support same named selections across children.
```

Comment:
No, that's not what I meant by this comment. Rather, if I have a selection named the same across multiple concatenated views, it should populate the same top-level state in Vega. I haven't yet reasoned through it all in my head, so I left the TODO in here to remind me once I've resolved the higher priority items.